changelog/upgrade-notes.md

```diff
@@ -2,6 +2,18 @@
 
 ### 9.2.0
 
+#### Flat styles: Removal of Type hints in `'get'` expressions
```

Comment:
Looks like this should go under a "next release" heading instead of 9.2.0.

Reply:
Fixed.